### ISSUE TITLE
Multi Instance User tasks - after rjsf update

### ISSUE THREAD

**Issue by madhurrya** [sartography/spiff-arena#1664]
Noticed that with the latest updates, now when we run a Multi instance user task (both parallel and sequential), the next user form shows the previous form's values. Earlier it didn't happen like that.
I don't know whether this is something that needs to be fixed or not.

Please check these 2 videos.
https://drive.google.com/file/d/1OvU08twwuMlYfPEwAk5Ohzlqj20brzNO/view?usp=sharing
https://drive.google.com/file/d/1EVHDo22FgHWYCLf2NfBKbUIRHaSjQlnf/view?usp=sharing

Or you can check with these models
https://dev.app.spiff.status.im/process-models/misc:qa:bpmn-model-testing:unit-test-multi-instance-sequential-user-task-loop-cardinality
https://test.app.spiff.status.im/process-models/misc:qa:bpmn-model-testing:unit-test-multi-instance-sequential-user-task-loop-cardinality

https://dev.app.spiff.status.im/process-models/misc:qa:bpmn-model-testing:unit-test-multi-instance-parallel-user-task-input-collection
https://test.app.spiff.status.im/process-models/misc:qa:bpmn-model-testing:unit-test-multi-instance-parallel-user-task-input-collection

**Comment by madhurrya** [comment]
Also noticed that in test.app we can submit the above forms without entering any data in the text field. But in dev.app when we do that it gives an error.

**Comment by burnettk** [comment]
for the second issue, where we used to be able to submit the form without entering any data, i think this is the same issue as that we experienced elsewhere where we no longer get task data keys when the value is blank. this is a change in react json schema form, and will require process model changes (which i think have largely already been completed for the important process models). so i think we can ignore that symptom in the context of this ticket.
